package.json

```diff
@@ -80,21 +80,17 @@
     "ts-jest": "^28.0.5",
     "tsc-files": "^1.1.3",
     "typechain": "^8.1.0",
-    "typescript": "^4.6.3"
+    "typescript": "^4.6.3",
+    "dts-bundle-generator": "^6.12.0",
+    "typedoc": "^0.23.10"
   },
   "dependencies": {
-    "@ethersproject/bignumber": "^5.6.2",
-    "@ethersproject/providers": "^5.6.4",
-    "@ethersproject/units": "^5.6.1",
-    "@ethersproject/wallet": "^5.6.2",
```

Comment:
question - does `ethers` provide more stuff than we need? Would it be better to remove `ethers` and only target the ones we actually use?

Reply:
Yes it does. Probably yes, although the concern is the same as this PR description states where we have to manage the versions of each module and being compatible with each other.